ng-dev/utils/git/git-client.ts

```diff
@@ -108,6 +114,13 @@ export class GitClient {
       process.stderr.write(this.sanitizeConsoleOutput(result.stderr));
     }
 
+    if (result.error !== undefined) {
```

Comment:
Am I correct in my understanding that `result.error` would be coming from the GitSanbox only?  Should we make a note that that specifically.

Reply:
It actually happens for any Git command (even without testing). It's a fix I basically sneaked into this PR as I noticed it while testing (which shows how valuable actual Git is for testing)